packages/inventory-compliance/src/SystemRulesTable/SystemRulesTable.js

```diff
@@ -172,6 +171,10 @@ class SystemRulesTable extends React.Component {
         });
     }
 
+    selectedRules = (rules) => this.addIsSelected(
+        this.state.selectedOnly ? rules.filter(rule => rule.isSelected) : rules
```

Comment:
Is this correct to call `addIsSelected` prior to filtering?

Reply:
It shouldn't affect any behavior. Filtering used to use `isSelected` but no longer needs it after this change.

Reply:
btw, `addIsSelected` is called _after_ `filteredRules`, not before.

Reply:
Lets rewrite it to this:
```
selectedRules = (rules) => {
    const filtered = this.state.selectedOnly ? rules.filter(rule => rule.isSelected) : rules;
    const selectAdded = this.addIsSelected(filtered);
    return selectAdded;
}
```

The `filtered` filters the rules by `isSelected` boolean which is added after the `filter()` is called.

What am I missing?

Reply:
Hmm... yes, I see your point :laughing: this does not seem correct at all.